rules/windows/process_creation/proc_creation_win_browsers_chromium_headless_file_download.yml

```diff
@@ -30,7 +30,9 @@ detection:
             - '--headless'
             - 'dump-dom'
             - 'http'
-    condition: selection
+    filter_main_legitimate_cli:
+        CommandLine|contains: '--headless --disable-gpu --disable-extensions --disable-plugins --mute-audio --no-first-run --incognito --aggressive-cache-discard --dump-dom'
```

Comment:
what triggert this? Not sure I want to add an exclusion, maybe we update only the falsepositives section. But we should name the culprit (main legitimate cli?) and maybe its an optional filter?